src/Widgets/calendar/ControlHeader.vala

```diff
@@ -25,28 +25,50 @@ namespace DateTime.Widgets {
         public signal void center_clicked ();
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
+
+            var label = new Gtk.Label (new GLib.DateTime.now_local ().format (_("%OB, %Y")));
+            label.get_style_context ().add_class (Granite.STYLE_CLASS_H2_LABEL);
+            label.halign = Gtk.Align.START;
+            label.set_size_request (185, -1);
```

Comment:
I think in this case, `width_chars` probably gives you more what you want and is agnostic of the text size in case the user makes it larger or smaller

```suggestion
            label.width_chars = 14;
```

Reply:
:+1: